entities/weapons/pocket/shared.lua

```diff
@@ -118,9 +118,14 @@ function SWEP:SecondaryAttack()
 end
 
 function SWEP:Reload()
-    if not CLIENT then return end
+    if CLIENT then
+      DarkRP.openPocketMenu()
```

Comment:
Won't this open the pocket menu twice on a server? Once through calling `DarkRP.openPocketMenu` here, and once through the net message received from the server?

Perhaps just add a `game.SinglePlayer()` check to sending the message?